include/spdk/bdev.h

```diff
@@ -133,6 +133,7 @@ enum spdk_bdev_io_type {
 	SPDK_BDEV_IO_TYPE_UNMAP,
 	SPDK_BDEV_IO_TYPE_FLUSH,
 	SPDK_BDEV_IO_TYPE_RESET,
+	SPDK_BDEV_IO_TYPE_PASSTHRU,
```

Comment:
Let's limit this PR to SPDK_BDEV_IO_TYPE_NVME_ADMIN only for now. I am not convinced of the need to provide a pass-through for IO, but we could add it later is needed.